Pipeline/CreateUCDComponentVersion/README.md

```diff
@@ -14,15 +14,17 @@ This section provides a more detailed explanation of how the CreateUCDComponentV
 
 1. **Initialization**
    1. Read command line parameters.
-   1. Read any application and global properties that are passed as a comma-separated list via `--packagingPropFiles` (This is optional for the UCD package format v1, but an required option using the UCD package format v2 - see `--ucdV2PackageFormat`).
+   1. Read any application and global properties files that are passed as a comma-separated list via `--packagingPropFiles`.
+       (This parameter is optional for UCD package format v1, but required when using UCD package format v2 - see `--ucdV2PackageFormat`.)
```

Comment:
If possible, we should provide a link when we say "see `--ucdV2PackageFormat`, so that readers can easily go to the more detailed documentation to learn more.

Reply:
ucdV2PackageFormat is a CLI option, we don't have any specific section talking about it.